### DISCUSSION TITLE
Reloading changed custom mappings without having to restart nvim

### DISCUSSION THREAD

**Discussion by t-ober** [NvChad/NvChad#2649] [Q&A] [answered]
So I am fairly new to the whole neovim game, maybe I have missed something.

I read that there were issues with full reloading in the past so now only the ui is "hot reloaded". 
Is there a way to reload changed custom mappings without restarting nvim?
Naive resourcing `custom/mappings.lua` does not seem to work.

**Comment by siduck** [comment]
`require("core.utils").load_mappings()` try that,  if it works then i'll include it in our autocmd

**Comment by t-ober** [comment]
That does not seem to quite work. It seems like the update keymapping is loaded but some other stuff breaks. Not least basic j / k file navigation, which give this output instead of navigating lines:

![image](https://github.com/NvChad/NvChad/assets/63147366/a90338f2-2cda-4e6d-951e-ccfe705bf834)

**Reply by siduck** [reply, marked as answer]
@t-ober ig its better not to reload  it then! Ive been told many a times  by one of the nvim core devs that we shouldnt reload our configs if its in lua
